RELEASE_NOTES.md

```diff
@@ -1,1326 +1,284 @@
-| `etcd` on `ncn-mxxx`         | 3.5.0   |
-| `ceph`                       | 16.2.13 |
-| `containerd`                 | 1.5.16  |
 | `coredns`                    | 1.8.4   |
-| `cray-certmanager`           | 1.5.5   |
-| `cray-spire`                 | 1.5.5   |
 | `helm`                       | 3.11.2  |
-| `istio`                      | 1.19.10 |
-| `keycloak`                   | 21.1.1  |
-| `kiali`                      | 1.75.0  |
-| `metrics-server`             | 0.6.3   |
-| `postgres-operator`          | 1.8.2   |
 | `postgresql`                 | 14.8    |
-| `spire-intermediate`         | 1.0.0   |
+| `postgres-operator`          | 1.8.2   |
 | `spire`                      | 0.12.2  |
-
-### Hardware Management System (HMS) improvements
-
-* More than 75% reduction in OS noise generated by the heartbeat client that runs on compute nodes.
-
-### Scale improvements
-
-* Performance improvements in BOS and CFS on large scale systems.
+| `spire-intermediate`         | 1.0.0   |
+| `cray-spire`                 | 1.5.5   |
+| `metrics-server`             | 0.6.3   |
+| `cray-certmanager`           | 1.5.5   |
+| `argo-workflows`             | 3.3.6   |
+| `argo-workflow-controller`   | 3.4.5   |
+| `ceph`                       | 16.2.13 |
 
 ### Security improvements
+TODO: Kyverno is old, is it sitll relevant?
 
 * `Kyverno` is upgraded from 1.9.5 version to 1.10.7 version to address CVEs.
+* CASMHMS-6282 Fix CVE's in artifactory.algol60.net/csm-docker/stable/cray-firmware-action:1.34.0
 
 ### Customer-requested enhancements
+TODO: Remove iSCSI unless more was added?
 
 * iSCSI based boot content projection for `rootfs` and `PE` images.
-
-### Resolved CASTs
-
-```text
-* CAST-26583 – ‘cray-bos’ gives an error if provided nids instead of xnames
-* CAST-27383 – Additional utilities have been provided for the NCN post-crash environment
-* CAST-32130 – The script to create RBD devices checks if RBD0 already exists
-* CAST-32623 – sat provides ability to generate longer and more complex BMC passwords through sat bmccreds
-* CAST-33693 - Unbound - interface used for NID alias is configurable
-* CAST-34561 – sat bootprep now supports functionality similar to the CFS Key ‘additional Inventory’
-* CAST-35618 – Fixed a python3 warning that blowfish is deprecated
-* CAST-36209 - "canu" user hanging around after install
-* CAST-36629 – Fixed an issue where ray-hms-smd-test-functional regex does not match RedfishSystemInfo/EthernetNICInfo/3/RedfishId
-* CAST-36655 – Fixed a failed regex test for sls in cray-hms-sls-test-functional
-* CAST-36752 – Fixed Check bos service test failure with templates that don't belong to expected tenant 'cmsdev-tenant'
-* CAST-36997 – inotify-tools have been added to the helm chart for cray-console operator
-* CAST-37016 – Fixed BOS sessions failing, nodes down when CFS component API is extremely slow
-```
+* CASMNET-2240 CSCS - CSM 1.5.1 canu generated switch config security concern
```

Comment:
CSM 1.5.1?

Reply:
https://jira-pro.it.hpe.com:8443/browse/CASMNET-2240 went into 1.6.1, the JIRA title has "1.5.1" for fixing an issue found in 1.5.1.